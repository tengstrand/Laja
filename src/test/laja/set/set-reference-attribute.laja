#write "target/test-output/set-reference-attribute.out"
  #set (myclass = new net.sf.laja.integrationtest.SetTest$Myclass())
  #set (map = { a:"a" b: { c: "hello" } })
  #set (map.a = "new value")
  #set (myclass.value = "hello")
map.a = {map.a}
myclass.value = {myclass.value}
map.b.c = {map.b.c}
  #set (map.b.c = "there")
map.b.c = {map.b.c}
#end
