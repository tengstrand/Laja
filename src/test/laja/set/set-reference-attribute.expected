map.a = new value
myclass.value = hello
map.b.c = hello
map.b.c = there
